README.md

```diff
@@ -63,7 +65,7 @@ $connector = new oliverlorenz\reactphpmqtt\Connector($loop, $resolver, $version)
 
 $p = $connector->create($config['server'], $config['port'], $config['options']);
 $p->then(function(\React\Stream\Stream $stream) use ($connector) {
-    $stream->on('PUBLISH', function(\oliverlorenz\reactphpmqtt\packet\Publish $message) {
+    $stream->on(Publish::EVENT, function(Publish $message) {
```

Comment:
Actually, if you think about it from the point of view of someone using the library, I think it makes sense like this because you can only listen for events for packets that have the `EVENT` constant. A user's IDE would hopefully tell them that they made a mistake if they tried to listen for an event on a different packet type.